deploy/pulse.conf

```diff
@@ -79,7 +89,6 @@ upstream staging_pulse {
 }
 
 server {
-  listen 443 ssl spdy;
```

Comment:
All of the `listen 443 ssl spdy` commands were removed, changed to `listen 443 ssl http2`, and moved into `ssl.rules` on the server. nginx was updated to `1.9.9`, and OpenSSL was updated to `1.0.2e`.